code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -824,6 +824,13 @@
 
 //Science
 
+/obj/item/circuitboard/machine/suit_storage
+	name = "Suit Storage Unit (Machine Board)"
+	greyscale_colors = CIRCUIT_COLOR_SCIENCE
```

Comment:
wouldnt it be engineering

Reply:
ah shit forgot i left it at science, thanks for catching it